docs/content/utilities/flexbox.md

```diff
@@ -487,44 +487,74 @@ When the main axis wraps, this creates multiple main axis lines and adds extra s
 Use this class to specify the ability of a flex item to alter its dimensions to fill available space.
 
 ```css
-.flex-auto       { flex: 1 1 auto; }
-.flex-grow-0     { flex-grow: 0; }
-.flex-shrink-0   { flex-shrink: 0; }
+.flex-1        { flex: 1; }
+.flex-auto     { flex: auto; }
```

Comment:
Changing from `flex: 1 1 auto;` to just `flex: auto;` should have no effect since the computed value is the same:

![Screen Shot 2019-10-31 at 8 00 39 PM](https://user-images.githubusercontent.com/378023/67941581-48f66500-fc19-11e9-9c4e-7e86e2bb7cff.png)